ably-ios/ARTAuth.m

```diff
@@ -201,11 +201,12 @@ - (void)executeTokenRequest:(ARTAuthTokenRequest *)tokenRequest callback:(void (
     }];
 }
 
-- (void)authorise:(ARTAuthTokenParams *)tokenParams options:(ARTAuthOptions *)options force:(BOOL)force callback:(void (^)(ARTAuthTokenDetails *, NSError *))callback {
+- (void)authorise:(ARTAuthTokenParams *)tokenParams options:(ARTAuthOptions *)options callback:(void (^)(ARTAuthTokenDetails *, NSError *))callback {
     BOOL requestNewToken = NO;
+    BOOL useTheForce = options ? options.force : false;
```

Comment:
useTheForce?  Sounds like something out of Star Wars.  Why not simply `forceTokenRenewal`?